libcaf_core/bii.cmake

```diff
@@ -0,0 +1,45 @@
+# build a shared library
+set(BII_LIB_TYPE SHARED)
```

Comment:
This forces biicode to create a dynamic shared library instead of its default static library.